configs/eslint-config-typescript/index.js

```diff
@@ -5,58 +5,62 @@ const ERROR = 2;
 module.exports = {
   root: true,
   extends: [
-    '@monkvision/eslint-config-base',
-    'plugin:@typescript-eslint/recommended',
-    'plugin:import/typescript',
+    "@monkvision/eslint-config-base",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:import/typescript",
   ],
-  plugins: ['@typescript-eslint'],
-  parser: '@typescript-eslint/parser',
+  plugins: ["@typescript-eslint"],
+  parser: "@typescript-eslint/parser",
   settings: {
-    'import/resolver': {
+    "import/resolver": {
       node: {
-        extensions: ['.js', '.jsx', '.ts', '.tsx'],
+        extensions: [".js", ".jsx", ".ts", ".tsx"],
       },
     },
   },
   rules: {
-    'prettier/prettier': ERROR,
-    '@typescript-eslint/lines-between-class-members': OFF,
-    'no-useless-constructor': OFF,
-    '@typescript-eslint/no-empty-function': [ERROR, { allow: ['arrowFunctions'] }],
-    '@typescript-eslint/no-empty-interface': OFF,
-    '@typescript-eslint/no-unused-vars': [
+    "prettier/prettier": ERROR,
+    "@typescript-eslint/lines-between-class-members": OFF,
+    "no-useless-constructor": OFF,
+    "@typescript-eslint/no-empty-function": [
+      ERROR,
+      { allow: ["arrowFunctions"] },
+    ],
+    "@typescript-eslint/no-empty-interface": OFF,
+    "@typescript-eslint/no-non-null-assertion": OFF,
```

Comment:
remove this rule : it should stay enabled

Reply:
i have few clarification on this.